saleor/plugins/avatax/plugin.py

```diff
@@ -203,17 +203,11 @@ def calculate_checkout_total(
         discounts: Iterable[DiscountInfo],
         previous_value: TaxedMoney,
     ) -> TaxedMoney:
-        if self._skip_plugin(previous_value):
-            return previous_value
-        checkout_total = previous_value
-
-        if not _validate_checkout(checkout_info, lines):
```

Comment:
Does it mean that we drop `_validate_checkout` from all plugin's methods?
If yes, what about checking if user provided shipping address? User address is mandatory to call Avatax request. Without it, I don't think that there is a sense to call Avatax - it will return an error (AFAIR).

Reply:
No, this is checked in `_get_checkout_tax_data` method.

Reply:
I do not see a reason to check it twice 😄 . The `_get_checkout_tax_data` is the helper that check both if the plugin is active and validates the checkout.